### PR TITLE
Feature/auth pages logic

### PR DESCRIPTION
**Pull request by OrionBetel**

### CODE REVIEW

File: frontend/src/store/modules/auth/mutations.js [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
 export default {
     SET_TOKEN: (state, token) => {
         state.token = token;
     },
     USER_LOGIN: (state, response) => {
-        StorageService.setToken(response.token);
-        state.token = response.token;
+        StorageService.setToken(response.access_token);
+        state.token = response.access_token;
         state.currentUser = response.user;
```
Comment (fadykstas) [inline frontend/src/store/modules/auth/mutations.js]: 'response' doesn't have 'user'

File: frontend/src/components/auth/Reset.vue [hunk 2]
```diff
@@ -35,8 +71,91 @@ export default {
...
+        },
+        passwordConfirm: {
+          type: ''
+        }
+      }
+    }
+  },
+
+  computed: {
+    // ...mapGetters(['getToken'])
```
Comment (lenchvolodymyr) [inline frontend/src/components/auth/Reset.vue]: don't leave any comments

File: frontend/src/store/modules/auth/mutations.js (hunk 1 above)
Comment (lenchvolodymyr) [reply frontend/src/store/modules/auth/mutations.js]: Yes, Stanislav rights, you need to call fetchAuthenticatedUser action after success login, like this:

```
 login: (context, user) => {
        httpService.post('auth/login', {
            email: user.email,
            password: user.password
        })
            .then((res) => {
                context.commit('USER_LOGIN', res.data.data);
                context.dispatch('fetchAuthenticatedUser', res.data.data.access_token);
            }).catch(function (err) {
                // TODO: Handle error
            });
    },
```

**Comment by lenchvolodymyr** [comment]
Add logout

**Pull request merged**